### PR TITLE
Update Geonetwork 3.8.0

### PR DESCRIPTION
**Pull request by elcoloo**
Hi @doublebyte1 is Ready update the Dockerfile for Geonetwork 3.8.0 & 3.8.0 with postgres. Is Ok?

### CODE REVIEW

File: 3.8.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM tomcat:8.5-jre8
```
Comment (juanluisrp) [inline 3.8.0/Dockerfile]: Please use `tomcat:8.5-jdk8` instead. 
See https://github.com/docker-library/tomcat/pull/158 to check the reasons.

**Comment by elcoloo** [comment]
Ready the change in the image of base.

File: 3.8.0/Dockerfile (hunk 1 above)
Comment (elcoloo) [reply 3.8.0/Dockerfile]: is Ready

**Comment by juanluisrp** [comment]
Thank you @elcoloo. Your help is much appreciated

**Pull request merged**